### ISSUE TITLE
Crash due to interface conversion in `apstra_datacenter_system` data source

### ISSUE THREAD

**Issue by chrismarget-j** [Juniper/terraform-provider-apstra#372]
```
363 Stack trace from the terraform-provider-apstra_v0.36.0 plugin:
364 panic: interface conversion: attr.Value is nil, not basetypes.StringValue
365 goroutine 769 [running]:
366 github.com/Juniper/terraform-provider-apstra/apstra.(*dataSourceDatacenterSystemNode).Read(0xc000606060, {0xfc4120, 0xc002cbf080}, {{{{0xfc9208, 0xc00361acc0}, {0xd30160, 0xc00361a960}}, {0xfcbf10, 0xc0006a3680}}, {{{0x0, ...}, ...}, ...}}, ...)
367 github.com/Juniper/terraform-provider-apstra/apstra/data_source_datacenter_system.go:49 +0x61e
```

**Comment by chrismarget-j** [comment]
cc: @fengwang523

**Comment by chrismarget-j** [comment]
Closed by #373
